common/shared.c

```diff
@@ -496,7 +491,6 @@ int ft_finalize(
 	msg.desc = NULL;
 	msg.iov_count = 1;
 	msg.addr = addr;
-	msg.context = &tx_ctx;
```

Comment:
Oh, we should check if the `PSM` provider stills passes the pingpnog tests with the changes I made. I think it might fail them, but I don't have any equipment to test with.

@shantonu Could you test this for me?